website/utils/url_utils.ts

```diff
@@ -748,6 +815,8 @@ export async function parseViewerUrlParams(urlSearchParams: URLSearchParams): Pr
     args = { ...args, ...datasetArgs };
   }
 
+  console.log("args", args);
+  console.log("viewer settings", viewerSettings);
```

Comment:
did you mean to leave these in here?

Reply:
Ah, I did not... Sorry for missing those!